app/src/main/java/com/stevesoltys/seedvault/ui/recoverycode/RecoveryCodeActivity.kt

```diff
@@ -15,7 +15,7 @@ class RecoveryCodeActivity : BackupActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        window.addFlags(FLAG_SECURE)
+        if (!isDebugBuild()) window.addFlags(FLAG_SECURE)
```

Comment:
CI failed. Maybe you need to import this?

Reply:
Done.